rhel7/profiles/stig.profile

```diff
@@ -157,6 +159,7 @@ selections:
     - grub2_enable_fips_mode
     - aide_verify_acls
     - aide_verify_ext_attributes
+    - aide_verify_ext_attributes.severity=low
```

Comment:
As this is by default, so the rule can change to match the expected severity.

Reply:
done